admin/test/features/assign_check_to_windows.feature

```diff
@@ -46,11 +46,13 @@ Feature:
     Given I am on the assign check window page
     Then check windows should have a end date
 
+  @wip
```

Comment:
@activemq  Shouldn't this be remove before merging?  Are these tests complete?

Reply:
@jon-shipley No this is basically these tests were working fine before this ticket was picked up. The BA's seemed to forgot to add AC's around assigning checks to a window, so now this functionality needs to added as a separate ticket....Thus the tests are marked @wip